.github/workflows/draft-pdf.yml

```diff
@@ -0,0 +1,24 @@
+name: Draft PDF
```

Comment:
Maybe just PDF?

Reply:
I just used the default setup recommended by JOSS.